sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmarkArguments.scala

```diff
@@ -29,7 +33,11 @@ class TPCDSQueryBenchmarkArguments(val args: Array[String]) {
     while(args.nonEmpty) {
       args match {
         case ("--data-location") :: value :: tail =>
-          dataLocation = value
+          dataLocation = value.toLowerCase(Locale.ROOT)
```

Comment:
Sorry, I mean the option name `--data-location` need to be case insensitive?

Reply:
oh! missed...

Reply:
`SparkSubmitArguments` handles options as case-sensitive, so better to make them case-insensitive, too?

Reply:
I am not sure about that one.

Reply:
ok